dotCMS/src/main/java/com/dotmarketing/portlets/templates/model/Template.java

```diff
@@ -24,6 +24,8 @@
 
 	private static final long serialVersionUID = 1L;
 
+	private static String ANONYMOUS_PREFIX = "anonymous_layout_";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "ANONYMOUS_PREFIX" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)